src/controllers/resetPassword.js

```diff
@@ -29,11 +31,13 @@ export default class ResetPasswordController {
       if (!user) {
         return errorResponse(res, 404, 'No user with email');
       }
-      const { id: userId, password: passwordHash, createdAt } = user;
+      const {
+        id: userId, password: passwordHash, createdAt, firstName
+      } = user;
       const secret = `${passwordHash}-${createdAt}`;
       const token = await Jwt.generateToken({ userId }, secret);
-      const url = `${req.headers.host}/api/v1/resetpassword/${userId}?token=${token}`;
-      // await sendMail(email, 'passwordRecovery', url);
+      const url = `${baseUrl}/forgotpassword/${userId}?token=${token}`;
+      await sendMail(email, 'passwordRecovery', {firstName, url});
```

Comment:
Is this `sendMail` or `sendEmail`? You're using `sendMail` and `sendEmail`. Please stick to one.